terraform/lib/dependabot/terraform/version.rb

```diff
@@ -31,6 +34,15 @@ def self.new(version)
       def to_s
         @version_string
       end
+
+      sig { override.params(version: VersionParameter).returns(T::Boolean) }
+      def self.correct?(version)
+        version = version.gsub(/^v/, "") if version.is_a?(String)
```

Comment:
these 2 lines are repeated in `initialize`; is this an opportunity to extract them out?

Reply:
Moved to a method